game/static/game/css/contributors.css

```
@@ -0,0 +1,573 @@
+  background: #3b82f6;
+
+  bottom: 0;
+  right: 10%;
+}
+
+/* CONTRIBUTORS */
+
+.contributors-section {
+  width: 100%;
+
+  padding: 2rem 40px 6rem;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)